docs/usage.rst

```diff
@@ -0,0 +1,103 @@
+.. include:: links.rst
+
+How to Use
+============
+
+Incorporating Eddymotion into a Python Module
+---------------------------------------------
+
+To utilize Eddymotion functionalities within your Python module, follow these steps:
+
+1. **Import Eddymotion Components**: Start by importing necessary components from the Eddymotion package:
+
+   .. code-block:: python
+
+      # Import required components from the Eddymotion package
+      from eddymotion.data.dmri import DWI
+      from eddymotion.estimator import EddyMotionEstimator
+
+2. **Load DWI Data**: Load diffusion MRI (dMRI) data into a `DWI` object using the `load` function. Ensure the gradient table is provided. It should have one row per diffusion-weighted image. The first three columns represent the gradient directions, and the last column indicates the timing and strength of the gradients in units of s/mm² [ R A S+ b ]. If your data are in NIfTI file format, include a file containing the gradient information with the argument "gradients_file":
+
+   .. code-block:: python
+
+      # Load dMRI data into a DWI object
+      dwi_data = DWI.load('/path/to/your/dwi_data.nii.gz', gradients_file='/path/to/your/gradient_file')
+
+3. **Instantiate an Eddymotion Estimator Object**: Create an instance of the `EddyMotionEstimator` class, which encapsulates tools for estimating rigid-body head motion and distortions due to eddy currents.
+
+   .. code-block:: python
+
+      # Create an instance of the EddyMotionEstimator class
+      estimator = EddyMotionEstimator()
+
+4. **Fit the Models to Estimate the Affine Transformation**:
+
+   Use the `fit` method of the `EddyMotionEstimator` object to estimate the affine transformation parameters:
+
+   .. code-block:: python
+
+      # Estimate affine transformation parameters
+      estimated_affine = estimator.fit(
```

Comment:
PR #173.

Reply:
Thanks for pointing out this change! Perhaps we could consider merging #173 before this PR (pinging @oesteban @effigies here), and then I can directly document the new method call in this PR

Reply:
Merged